saml-core-api/src/main/java/org/keycloak/saml/common/util/Base64.java

```diff
@@ -100,31 +101,9 @@
     /** The new line character (\n) as a byte. */
     private static final byte NEW_LINE = (byte) '\n';
 
-    /** Preferred encoding. */
-    private static final String PREFERRED_ENCODING = "UTF-8";
-
     /** The 64 valid Base64 values. */
-    private static final byte[] ALPHABET;
-    private static final byte[] _NATIVE_ALPHABET = /* May be something funny like EBCDIC */
-            {(byte) 'A', (byte) 'B', (byte) 'C', (byte) 'D', (byte) 'E', (byte) 'F', (byte) 'G', (byte) 'H', (byte) 'I', (byte) 'J',
-                    (byte) 'K', (byte) 'L', (byte) 'M', (byte) 'N', (byte) 'O', (byte) 'P', (byte) 'Q', (byte) 'R', (byte) 'S',
-                    (byte) 'T', (byte) 'U', (byte) 'V', (byte) 'W', (byte) 'X', (byte) 'Y', (byte) 'Z', (byte) 'a', (byte) 'b',
-                    (byte) 'c', (byte) 'd', (byte) 'e', (byte) 'f', (byte) 'g', (byte) 'h', (byte) 'i', (byte) 'j', (byte) 'k',
-                    (byte) 'l', (byte) 'm', (byte) 'n', (byte) 'o', (byte) 'p', (byte) 'q', (byte) 'r', (byte) 's', (byte) 't',
-                    (byte) 'u', (byte) 'v', (byte) 'w', (byte) 'x', (byte) 'y', (byte) 'z', (byte) '0', (byte) '1', (byte) '2',
-                    (byte) '3', (byte) '4', (byte) '5', (byte) '6', (byte) '7', (byte) '8', (byte) '9', (byte) '+', (byte) '/'};
-
-    /** Determine which ALPHABET to use. */
-    static {
-        byte[] __bytes;
-        try {
-            __bytes = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/".getBytes(PREFERRED_ENCODING);
-        } // end try
-        catch (java.io.UnsupportedEncodingException use) {
-            __bytes = _NATIVE_ALPHABET; // Fall back to native encoding
-        } // end catch
-        ALPHABET = __bytes;
-    } // end static
+    private static final byte[] ALPHABET =
+            "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/".getBytes(StandardCharsets.UTF_8);
```

Comment:
Please remove this change. This is not correct in this context

Reply:
@hmlnarik Done :)